src/rt/object/object.h

```diff
@@ -140,7 +140,7 @@ namespace verona::rt
       case EpochMark::SCANNED:
         return os << "SCANNED";
       default:
-        return os;
+        abort();
```

Comment:
Nit: this change is not described in the commit message

Reply:
This is consistent with a previous decision to abort on a missed case in the stream operator overload for `BackpressureState`. This would have caught the error of logging the garbage epoch mark of the token cown before prerun.